src/assets/ts/main.ts

```diff
@@ -122,8 +122,18 @@ export function getWebAsset(assetId: string | null, user: User) {
   )
 }
 
-export function getAssetDashboardLink(assetId: string) {
-  return `https://login.screenlyapp.com/login?next=/manage/assets/${assetId}`;
+export function getTeamInfo(user: User, teamId: string) {
+  const queryParams = `id=eq.${encodeURIComponent(teamId || '')}`;
+  return callApi(
+    'GET',
+    `https://api.screenlyapp.com/api/v4.1/teams/?${queryParams}`,
```

Comment:
```suggestion
    `https://api.screenlyapp.com/api/v4/teams/?${queryParams}`,
```

Reply:
@rusko124, I'll address this in the future as doing GET to `/api/v4/teams/?<query_params>` gives me a **_404_**.